BeagleBone_black/src/motor_control/controller.cpp

```diff
@@ -28,22 +28,88 @@
 namespace hyped {
 namespace motor_control {
 
-Controller::Controller(Logger& log)
+using hyped::utils::io::can::Frame;
+
+constexpr uint16_t sdo_receive = 0x600;
+
+// Common CAN messages for all motor Controllers
+// Device Control Command messages for all controllers
+// pg49 CANOpen_Motion_Control.pdf
+// TODO(anyone) look over again
+constexpr Frame SHUTDOWN            = {sdo_receive, false, 8, {0x2B, 0x60, 0x40, 0, 0x00, 0x06, 0, 0}}; //NOLINT
```

Comment:
See [naming of constants](https://hyp-ed.github.io/styleguide/#Constant_Names) in the style guide.